### ISSUE TITLE
Meross device pairing error (com.meross.meross)

### ISSUE THREAD

**Issue by FauconSpartiate** [AdguardTeam/AdguardForAndroid#4989]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

V4.1

### Environment

```markdown
- OS: Android 13
- Device: Oneplus 8T
- Firmware: KB2003_13.1.0.581(EX01)
```

### Root access

- [ ] Yes, I have it.

### What filters do you have enabled?

No filters enabled

### What Stealth Mode options do you have enabled?

/

### Issue Details

Steps to reproduce:
- Open app
- Add new device
- Select device that does pairing over WiFi and not bluetooth

### Expected Behavior

Pairing succeeds flawlessly

### Actual Behavior

Setup fails unless AdGuard is turned off

### Screenshots

<details><summary>Screenshot 1:</summary>

![image](https://github.com/AdguardTeam/AdguardForAndroid/assets/49079695/afceef3e-9a54-4950-bf19-a6f6ea73b29e)

</details>

### Additional Information

Original issue in AdguardFilters repository: https://github.com/AdguardTeam/AdguardFilters/issues/160557

**Comment by maxikuzmin** [comment]
@FauconSpartiate any news? I didn't get your logs in the mail

**Comment by FauconSpartiate** [comment]
@maxikuzmin I don't have the device near me right now, I'll do it as soon as I do.

**Comment by maxikuzmin** [comment]
@FauconSpartiate any news?

**Comment by FauconSpartiate** [comment]
@maxikuzmin I just sent the logs per email

**Comment by maxikuzmin** [comment]
@FauconSpartiate we need some more time. Please wait

**Comment by maxikuzmin** [comment]
@FauconSpartiate does the app filtering exception help you? 
Go to App management -> Merross app -> Route traffic through AdGuard -> off switch

**Comment by maxikuzmin** [comment]
@FauconSpartiate could you temporarily remove the exception for the Merross app, repeat the steps, and see what requests are blocked in the filtering log? Please send screenshots to [devteam@adguard.com](mailto:devteam@adguard.com). Blocked requests should be opened and screenshot the detailed information of the blocked request

**Comment by FauconSpartiate** [comment]
@maxikuzmin
Sorry for the delay.
I've just tried, and don't see anything obvious. The only requests that get blocked are ones to a stat server, but these occur also when having the app open normally, not while trying to pairthe device.
I also tried overriding this rule and still allowing it, but it didn't fix the bug.

So disabling filtering entirely doesn't solve the problem, but disabling the routing through Adguard does.
So it's probably a problem with the path the packets take, not the filtering having some false positives...
